unlock-app/src/__tests__/utils/withConfig/__snapshots__/missingProvidersWithConfig.test.js.snap

```diff
@@ -0,0 +1,119 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
Hoooo. interesting, you're using snapshot. Normally I would ask for consistency in the tests, but since I want us to move to snapshots here too, I'll accept that and see it as a nudge to get these done in the rest of the code.

Reply:
I am glad you are planning to move to use snapshots whenever you can. The shallow and find didn't play well with this test, so I decided to go with snapshots, it's easier to write and understand.